<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>709</width>
    <height>733</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Whale File Manager</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 14pt &quot;Sans Serif&quot;;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Rapid Access:</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
        </property>
        <item>
         <widget class="QPushButton" name="pushButton_addcustom">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>41</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>41</height>
           </size>
          </property>
          <property name="text">
           <string>+</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_custom_delete">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>41</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>41</height>
           </size>
          </property>
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_save_custom">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>41</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset theme="QIcon::ThemeIcon::DocumentSave"/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_custom_refresh">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>41</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset theme="QIcon::ThemeIcon::ViewRefresh"/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QListWidget" name="listWidget_custom">
        <property name="minimumSize">
         <size>
          <width>256</width>
          <height>217</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>256</width>
          <height>217</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_libraries">
        <property name="text">
         <string>Libraries:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="listWidget_libs">
        <property name="minimumSize">
         <size>
          <width>256</width>
          <height>218</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>256</width>
          <height>218</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_devices">
        <property name="text">
         <string>Devices:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="listWidget_disks">
        <property name="minimumSize">
         <size>
          <width>256</width>
          <height>217</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>256</width>
          <height>217</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="sizeConstraint">
       <enum>QLayout::SizeConstraint::SetMaximumSize</enum>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="lineEdit_path">
          <property name="text">
           <string>Path</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_go">
          <property name="text">
           <string>GO</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="pushButton_make_folder">
          <property name="text">
           <string>MAKE FOLDER</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_make_file">
          <property name="text">
           <string>MAKE FILE</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_search">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="inputMask">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>Search</string>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="listWidget_files">
        <property name="mouseTracking">
         <bool>false</bool>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::ContextMenuPolicy::CustomContextMenu</enum>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="defaultDropAction">
         <enum>Qt::DropAction::MoveAction</enum>
        </property>
        <property name="viewMode">
         <enum>QListView::ViewMode::ListMode</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>RENAME</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QPushButton" name="pushButton_move">
          <property name="text">
           <string>Move</string>
          </property>
          <property name="icon">
           <iconset theme="QIcon::ThemeIcon::GoNext"/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_pastemove">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Paste move</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="pushButton_copy">
          <property name="text">
           <string>Copy</string>
          </property>
          <property name="icon">
           <iconset theme="QIcon::ThemeIcon::EditCopy"/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_paste">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Paste</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>DELETE</string>
        </property>
        <property name="icon">
         <iconset theme="QIcon::ThemeIcon::EditDelete"/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <action name="actionCopy">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::EditCopy"/>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
